set.mm

```diff
@@ -661385,64 +661469,110 @@ D Fn ( I ... ( M - 1 ) ) /\
       TVAURIKVCVFVGVDULUNUO $.
   $}
 
+  $( Reverse closure for a sub-division-ring predicate.  (Contributed by SN,
+     19-Feb-2025.) $)
+  sdrgrcl $p |- ( A e. ( SubDRing ` R ) -> R e. DivRing ) $=
+    ( csdrg cfv wcel cdr csubrg cress co issdrg simp1bi ) ABCDEBFEABGDEBAHIFEBA
+    JK $.
+
+  ${
+    sdrgdrng.1 $e |- S = ( R |`s A ) $.
+    $( A sub-division-ring is a division ring.  (Contributed by SN,
+       19-Feb-2025.) $)
+    sdrgdrng $p |- ( A e. ( SubDRing ` R ) -> S e. DivRing ) $=
+      ( csdrg cfv wcel cress co cdr csubrg issdrg simp3bi eqeltrid ) ABEFGZCBAH
+      IZJDOBJGABKFGPJGBALMN $.
+  $}
+
+  ${
+    sdrgbas.b $e |- S = ( R |`s A ) $.
+    $( Base set of a sub-division-ring structure.  (Contributed by SN,
+       19-Feb-2025.) $)
+    sdrgbas $p |- ( A e. ( SubDRing ` R ) -> A = ( Base ` S ) ) $=
+      ( csdrg cfv wcel cbs wss wceq eqid sdrgss ressbas2 syl ) ABEFGABHFZIACHFJ
+      OBAOKZLAOCBDPMN $.
+  $}
+
+  $( A sub-division-ring is a subring.  (Contributed by SN, 19-Feb-2025.) $)
+  sdrgsubrg $p |- ( A e. ( SubDRing ` R ) -> A e. ( SubRing ` R ) ) $=
+    ( csdrg cfv wcel cdr csubrg cress co issdrg simp2bi ) ABCDEBFEABGDEBAHIFEBA
+    JK $.
```

Comment:
Those theorems are likely to shorten several proofs involving sub-division-rings.
I think they should be moved to main, and a minimization could be run with them.

Reply:
(not necessarily in this PR)